### PR TITLE
Fix Hangouts badge counter

### PR DESCRIPTION
**Pull request by tamas646**
#### Pre-flight Checklist

Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.

#### Description of Change

Fixed the counting of the notifications in the Hangouts recipe.

### CODE REVIEW

File: recipes/hangouts/webview.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 module.exports = Ferdium => {
   const getMessages = () => {
     // get unread messages
-    const count = document.querySelector('#hangout-landing-chat iframe').contentWindow.document.querySelectorAll('.ee').length;
+    let count = 0;
+    for (const span of document.querySelectorAll('span[jsname=DW2nlb]'))  count += span.textContent == '' ? 0 : Number.parseInt(span.textContent);
```
Comment (vraravam) [inline recipes/hangouts/webview.js:5]: Please use `Ferdium.safeParseInt` which will remove the need for the ternary null/blank check.

File: recipes/hangouts/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "id": "hangouts",
   "name": "Hangouts",
-  "version": "1.3.0",
+  "version": "1.4.0",
```
Comment (vraravam) [inline recipes/hangouts/package.json:4]: please bump only the tiny segment (3rd segment) for this change

**Review by vraravam — CHANGES_REQUESTED** [review]

**Comment by tamas646** [comment]
I've made the requested changes.

**Review by vraravam — APPROVED** [review]

**Comment by vraravam** [comment]
I dont use hangouts chat recipe, so I haven't tested this locally. but the changes look good.

**Comment by vraravam** [comment]
@all-contributors please add @tamas646 for code

**Pull request merged**